cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -0,0 +1,283 @@
+import sys
```

Comment:
Rook header.

Reply:
Why we require a Rook header (if we are not using it)

Reply:
For licensing.

Reply:
Oh! Ok licencing header, I thought of 'header-files' which should be imported from rook... (=
Added... Please check...